tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py

```diff
@@ -0,0 +1,254 @@
+        experiment_name=experiment_name,
+        cluster_name=cluster_name,
+        performance_profile=performance_profile,
+        mode=mode,
+        target_cluster=target_cluster,
+        datasource=datasource,
+        experiment_type=experiment_type,
+        kubernetes_obj_type=kubernetes_obj_type,
+        name=name,
+        namespace=namespace,
+        namespace_name=namespace_name,
+        container_image_name=container_image_name,
+        container_name=container_name,
+        measurement_duration=measurement_duration,
+        threshold=threshold
+    )
+
+    # Convert rendered content to a dictionary
+    json_content = json.loads(content)
+
+    if json_content[0]["kubernetes_objects"][0]["type"] == "None":
+        json_content[0]["kubernetes_objects"][0].pop("type")
+    if json_content[0]["kubernetes_objects"][0]["name"] == "None":
+        json_content[0]["kubernetes_objects"][0].pop("name")
+    if json_content[0]["kubernetes_objects"][0]["namespace"] == "None":
+        json_content[0]["kubernetes_objects"][0].pop("namespace")
+    if json_content[0]["kubernetes_objects"][0]["containers"][0]["container_image_name"] == "None":
+        json_content[0]["kubernetes_objects"][0].pop("containers")
+    if json_content[0]["kubernetes_objects"][0]["namespaces"]["namespace_name"] == "None":
+            json_content[0]["kubernetes_objects"][0].pop("namespaces")
+    if json_content[0]["kubernetes_objects"][0]["experiment_type"] == "None":
+        json_content[0]["kubernetes_objects"][0].pop("experiment_type")
+
+    # Write the final JSON to the temp file
+    with open(tmp_json_file, mode="w", encoding="utf-8") as message:
+        json.dump(json_content, message, indent=4)
+
+    input_json_file = tmp_json_file
+    form_kruize_url(cluster_type)
+
+    response = delete_experiment(input_json_file)
+    print("delete exp = ", response.status_code)
+
+    # Create experiment using the specified json
+    response = create_experiment(input_json_file)
+
+    data = response.json()
+    print(data['message'])
+
+    assert response.status_code == expected_status_code
+    assert data['status'] == ERROR_STATUS
+
+    response = delete_experiment(input_json_file)
+    print("delete exp = ", response.status_code)
+
+
+@pytest.mark.negative
+def test_create_multiple_namespace_exp(cluster_type):
+    """
+    Test Description: This test validates the response status code of createExperiment API
+    if multiple entries are presnet in create experiment json
+    """
+    input_json_file = "../json_files/create_multiple_namespace_exp.json"
+    form_kruize_url(cluster_type)
+
+    response = delete_experiment(input_json_file)
+    print("delete exp = ", response.status_code)
+
+    # Create experiment using the specified json
+    response = create_experiment(input_json_file)
+
+    data = response.json()
+    print(data['message'])
+
+    assert response.status_code == ERROR_STATUS_CODE
+    assert data['status'] == ERROR_STATUS
```

Comment:
Is there an error message returned here? This needs to be validated against the expected error message

Reply:
Added the validation for expected error message.